runtime/kusama/src/weights/pallet_babe.rs

```diff
@@ -0,0 +1,52 @@
+// Copyright 2017-2020 Parity Technologies (UK) Ltd.
+// This file is part of Polkadot.
+
+// Polkadot is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Polkadot is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Default weights for the Babe Pallet
+//! This file was not auto-generated.
+
+use frame_support::{
+	traits::Get,
+	weights::{
+		Weight,
+		constants::{WEIGHT_PER_MICROS, WEIGHT_PER_NANOS},
+	}
+};
+use sp_std::marker::PhantomData;
+
+pub struct WeightInfo<T>(PhantomData<T>);
+impl<T: frame_system::Trait> pallet_babe::WeightInfo for WeightInfo<T> {
```

Comment:
This is copy past from Substrate. @andresilva 

Wouldnt it be better just to pick this up from Substrate? Since we cannot measure this on its own in Polkadot.

Reply:
Yeah I agree it's not worth it to just copy paste this. Since this file is not auto-generated we can instead add a comment on it in Substrate to note that it is currently used in Polkadot. The original concern here was that any usage of a default weight could get changed unexpectedly.